static/oas/v1/sports/stats/goals.yml

```diff
@@ -30,7 +30,7 @@ paths:
             schema:
               $ref: "#/components/schemas/GoalRequest"
       responses:
-        200:
+        201:
```

Comment:
Again why 201?

Reply:
201 is the status code for Created, which is correct for Post requests